_posts/2025-02-10-bash_scripting_an_overview.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title:  "Getting Started with Bash scripting"
+description: Linux
+date:   2025-01-22 18:57:00 +0300
+image:  '/images/blogs/rebecca_Intro_to_bash_scripting_square.png'
+tags:   [linux, bash, opensource]
+author: Reshama Shaikh
+---
+
```

Comment:
@SangamSwadiK Add your name as the author.

Reply:
Seems like adding my name to author isnt being recognized

Reply:
@SangamSwadiK The author in the top header does not seem to be working yet. Still needs tweaking, so we'll keep it in the body of the text.